### PR TITLE
Switch to git based repository in OBS

### PR DESCRIPTION
**Pull request by davidcassany**
Part of rancher/elemental-obs#18

Essentially renaming some files and folders to better match OBS structure and including `_service` files where needed.

### CODE REVIEW

**Comment by davidcassany** [comment]
e2e tests will fail as they rely on a longer valid OBS build for PRs which won't be happening anymore after merging this. I'd like to track elemental-operator e2e tests in a separate issue. After moving Dev to git, e2e tests on PRs should be reconsidered.

File: .obs/specfile/elemental-operator/elemental-operator.spec [hunk 1]
```diff
@@ -15,6 +15,8 @@
 # Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
+%define commit _replaceme_
```
Comment (fgiudici) [inline .obs/specfile/elemental-operator/elemental-operator.spec:18]: are the "_replaceme_" managed in OBS ? Guess so, but worth double checking... 😄

**Review by fgiudici — APPROVED** [review]
Not fully clear to me yet, skimmed through the changes and LGTM!
Change of the charts name, if we will decide then to do so, to come later, right?

File: .obs/specfile/elemental-operator/elemental-operator.spec (hunk 1 above)
Comment (davidcassany) [reply .obs/specfile/elemental-operator/elemental-operator.spec:18]: Not really by OBS, they are managed by the scripts in rancher/elemental-obs that pull this code. But yes, I should add a comment here so this is explicit, I think I did that already for the toolkit, should the same here.

Comment (fgiudici) [reply .obs/specfile/elemental-operator/elemental-operator.spec:18]: well, was just a curiosity, I think you can merge as is 👍🏼

**Pull request merged**